packages/flame_bloc/lib/src/flame_bloc_game.dart

```diff
@@ -132,3 +128,10 @@ class FlameBlocGame extends FlameGame {
     });
   }
 }
+
+/// Provides a default, concrete implementation of a [FlameBloc] game.
+class FlameBlocGame extends FlameGame with FlameBloc {
```

Comment:
is this still necessary? or just for backwards compat?

Reply:
It serves as both for backwards compatibility and as a shortcut.